app/views/entitlements/index.html.erb

```diff
@@ -17,7 +17,7 @@
       <div id="entitlements-search" hidden>
         <p>Enter your rank, dependency status, and move type to determine your moving allowance.</p>
 
-        <%= form_with url: entitlements_path, class: 'usa-form', method: 'get' do |form| %>
+        <%= form_with url: entitlements_path, class: 'usa-form', method: 'get', skip_enforcing_utf8: true do |form| %>
```

Comment:
This addition (and the two additions in `app/views/offices/index.html.erb` will remove the `<input type="hidden" name="utf8" value="✓">` appended by `form_with`.